README.md

```diff
@@ -130,7 +144,16 @@ pip install -r requirements_optional.txt
 cd doc; make clean; make html
 ```
 
-## Support
+## About Us
+
+### Core contributors
+
+We are a team of students and researchers at various academic institutions: Berkeley, CNRS, Cornell, ENPC, ENS, Flatiron Institute, NYU, U of Montreal, and WWU; to name a few.
+
+You can find the full list of contributors on [GitHub insights](https://github.com/kymatio/kymatio/graphs/contributors).
```

Comment:
Why not just `[here]`? Will GH kick us off if we don't repeat their branding?

Reply:
Principle of least surprise. I don't really like links which say "here". But if you have a better description, i'm happy to hear it.

Reply:
What about `For more information, please see the [full list of contributors](...)`?